### PR TITLE
Minor updates to the Packages docs

### PR DESCRIPTION
**Pull request by sofietoft**
- Added more details about package migrations to the "Creating a package" article.
- Updated version numbers on the UmbPack article.
- Added note to UmbPack article about how to use NuGet instead.

### CODE REVIEW

File: Extending/Packages/Creating-a-Package/index.md [hunk 1]
```diff
@@ -296,6 +300,8 @@ Whichever migration plan you choose to create, you will be able to see that your
 
 ![Installed package](images/installed-package.png)
 
+When using a custom package migration plan, any changes to the plan will trigger all migrations again even those that have already run previously. This is due to the `IgnoreCurrentState` being set to `true` when changes are made to the plan. You can override this property and set it to `false` again to make it behave like regular migration plans and only run the migrations that have not yet been executed on the current environment.
```
Comment (ronaldbarendse) [inline Extending/Packages/Creating-a-Package/index.md:303]:
```suggestion
When using a custom package migration plan, the current state is ignored by default, causing it to execute all migrations again whenever this isn't the same as the final state of the plan (e.g. if you added a new migration). This is due to the `IgnoreCurrentState` being set to `true` in the `PackageMigrationPlan` base class. You can override this property and set it to `false` again to make it behave like regular migration plans and only run the migrations that have not yet been executed on the current environment.
```

File: Extending/Packages/UmbPack/index.md [hunk 2]
```diff
@@ -10,11 +11,17 @@ meta.Description: "How to use the UmbPack tool to deploy package versions to Our
 UmbPack is an open source NuGet tool that can deploy packages to Our, and also help you pack your files into a package.
 
 :::note
-You still need to create the package on Our before the tool can be used. Once it has been set up you will then be able to add new versions of the package zip files using UmbPack
+**Are you using Umbraco 9+?**
+
+As of Umbraco 9 all packages needs to be installed through NuGet, and it is no longer to install packages from .zip files.
```
Comment (ronaldbarendse) [inline Extending/Packages/UmbPack/index.md:16]: Should this also reference the blogpost (https://umbraco.com/blog/packages-in-umbraco-9-via-nuget/) explaining this change?

```suggestion
As of Umbraco 9, packages can only be installed using NuGet and it is no longer possible to install packages from package zip files.
```

**Review by ronaldbarendse — CHANGES_REQUESTED** [review]
Hi @sofietoft! I've read through the changes and only have two minor suggestions... Great additions and will hopefully safe people some time figuring out why their migrations are executed more than once👍🏻

**Comment by sofietoft** [comment]
Thanks for the suggestions @ronaldbarendse ! 💪 
I've applied them both - but did make a small adjustment to shorten the sentence length a bit 😄 

From:
When using a custom package migration plan, **the current state is ignored by default, causing it to execute all migrations** again whenever this isn't the same as the final state of the plan (e.g. if you added a new migration).
To:
When using a custom package migration plan, **the current state is ignored by default. This causes it to execute all migrations** again whenever this isn't the same as the final state of the plan (e.g. if you added a new migration).

**Pull request merged**